ecosystem/sep-0006.md

```diff
@@ -768,6 +772,22 @@ Example response:
       "completed_at": "2017-03-20T17:09:58Z",
       "stellar_transaction_id": "17a670bc424ff5ce3b386dbfaae9990b66a2a37b4fbe51547e8794962a3f9e6a",
       "external_transaction_id": "2dd16cb409513026fbe7defc0c6f826c2d2c65c3da993f747d09bf7dafd31093"
+    },
+    {
+      "id": "52fys79f63dh3v1",
+      "kind": "withdrawal",
+      "status": "pending_transaction_info_update",
```

Comment:
One quick question, why not change back to `incomplete`?

Reply:
> why not change back to `incomplete`?

That could work, but using a new status allows us to indicate that the transaction previously transitioned away from `incomplete`.

> What's the value in a separate status for these cases?

The client could deduce what is necessary by checking `required_info_updates` and if it were null/empty checking SEP-12's `GET /customer`, but this is more explicit.

If you see a strong reason to change the way this is designed we can do it, but since this uses the same pattern as SEP-31 I think there is value in being consistent across protocols.

Reply:
If it's the same pattern as SEP-31 I think that's a strong reason to do this. 

I don't think there is an always best approach to less or more states. Reusing states when they essentially mean the same thing can make some APIs simpler while making others more complex. APIs that never revisit states can't be easier to debug and code for, even if it means understanding then required more effort.

I'm still for it 👍